deploy/crds/planetscale_v2_vitessshard_crd.yaml

```diff
@@ -679,6 +679,35 @@ spec:
               items:
                 type: string
               type: array
+            conditions:
+              additionalProperties:
+                properties:
+                  lastTransitionTime:
+                    description: Last time the condition transitioned from one status
+                      to another. Optional.
+                    format: date-time
+                    type: string
+                  message:
+                    description: Human-readable message indicating details about last
+                      transition. Optional.
+                    type: string
+                  reason:
+                    description: Unique, one-word, CamelCase reason for the condition's
+                      last transition. Optional.
+                    type: string
+                  status:
+                    description: Status is the status of the condition. Can be True,
+                      False, Unknown.
+                    enum:
+                    - True;False;Unknown
```

Comment:
FYI this is how I can tell that semicolons don't work in this version of the generator. Each of these should be its own list item in the generated output.

Reply:
Got it, will update.

Reply:
Updated https://github.com/planetscale/vitess-operator/pull/31/commits/20698920e71c38d09894ba8b1dfadbe395d879d26.